doc/nrf/known_issues.rst

```diff
@@ -1185,6 +1185,20 @@ NCSDK-13923: Device may crash during Bluetooth bonding
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``4870fcd8316bd3a4b53ca0054f0ce35e1a8c567d``).
 
+.. rst-class:: v2-1-0
+
+NCSDK-16644: Machine learning application does not sleep on Thingy:53
```

Comment:
Do we need a separate issue for CAF sensor manager? (the module can be used outside of ML app)

btw. Maybe we could combine these 2 as `power management issues on Thngy:53` ? Users would not actually replicate wake up issue, because their Thingy:53 would never go to sleep.